.github/workflows/Tests.yml

```
@@ -1,5 +1,7 @@
 ---
 name: CI-TESTS
+description: "Continuous Integration for post-MATs additional testing, and validation."
```

actionlint (1.7.7)

unexpected key "description" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)